### PR TITLE
Pull Request for the branch (todo crud)

### PR DESCRIPTION
**Pull request by HamzaAnwar1998**
In this milestone, I have added functionalities

1. Create todo
2. Read todo
3. Update todo
4. Delete todo

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,26 +1,120 @@
...
+    let todos;
+    if (localStorage.getItem('Todos') !== null) {
+      todos = JSON.parse(localStorage.getItem('Todos'));
+    } else {
+      todos = [];
+    }
+    return todos;
+  }
+
+  // adding todo to local storage
```
Comment (dasileker) [inline src/index.js]: - [ ] Please remove the disabled linters and fix all the issues related to your code like you have too many classes, you should have just 1 as the linter set it.

File: src/modules/deleteTodoFromLS.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    const filteredTodos = storedTodos.filter((storedTodo) => storedTodo.ID !== idInNum);
+    for (let i = 0; i < filteredTodos.length; i += 1) {
+      filteredTodos[i].ID = filteredTodos.indexOf(filteredTodos[i]) + 1;
+    }
+    localStorage.setItem('Todos', JSON.stringify(filteredTodos));
+  }
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export { deleteTodoFromLS };
```
Comment (dasileker) [inline src/modules/deleteTodoFromLS.js]: - [ ] Also please use the export default instead of just export.

File: src/modules/deleteTodoFromUI.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const deleteTodoFromUI = (el) => {
+  if (el.classList.contains('remove-btn')) {
+    el.parentElement.parentElement.remove();
+  }
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export { deleteTodoFromUI };
```
Comment (dasileker) [inline src/modules/deleteTodoFromUI.js]: - [ ] Also please use the export default instead of just export.

File: src/modules/addTodoToUI.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    <div class="right">
+      <span class="fa-solid fa-trash fa-lg elippse-icon remove-btn">
+      </span>
+    </div>
+  </li>      
+        `;
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export { addTodoToUI };
```
Comment (dasileker) [inline src/modules/addTodoToUI.js]: - [ ] Also please use the export default instead of just export.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @HamzaAnwar1998,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**

- Linters are passing 👌.
- The PR is good 👌.
- Good readme file 👌.


## Required Changes ♻️
_Check the comments under the review._
- Nice job on your To-Do list app however, you have disabled too many linters you should not disable them all just one and the `package.json` is missing the start option to lunch the app.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HamzaAnwar1998** [comment]
- npm run dev is changed to npm start
- only one class is used in the index.js file
- classes has been changed to functions
- export {file} is changed to export default file

**Comment by HamzaAnwar1998** [comment]
The GitHub checks were running for the first time but now they are not running, I think that the checks are down following the recent trend. Please see this slack thread https://microverse-students.slack.com/archives/GA812K2TC/p1648052609972159?thread_ts=1648052415.193529&cid=GA812K2TC

**Pull request merged**